dev-tools/packaging/templates/docker/Dockerfile.elastic-agent.tmpl

```diff
@@ -70,15 +70,13 @@ RUN apt-get update -y && \
         libasound2\
         libatspi2.0-0\
         libxshmfence1 \
-        fonts-arphic-ukai \
-        fonts-arphic-uming \
-        fonts-ipafont-mincho \
-        fonts-ipafont-gothic \
-        fonts-unfonts-core && \
+        fonts-noto-core\
+        fonts-noto-cjk &&\
         apt-get clean all && \
         exit_code=0 && break || exit_code=$? && echo "apt-get error: retry $iter in 10s" && sleep 10; \
     done; \
     (exit $exit_code)
+RUN apt install fonts-noto-cjk
```

Comment:
Why do we explicitly install the fonts on elastic-agent image and not on the HB image?

Reply:
ah, that's a bug, it's actually installed higher up, this is redundant